app/src/lib/editors/win32.ts

```diff
@@ -498,6 +498,14 @@ const editors: WindowsExternalEditor[] = [
     displayNamePrefixes: ['Pulsar'],
     publishers: ['Pulsar-Edit'],
   },
+  {
+    name: 'JetBrains RustRover',
+    registryKeys: registryKeysForJetBrainsIDE('RustRover'),
```

Comment:
The `registryKeysForJetBrainsIDE` automatically generates possible registry keys for major and minor stable versions of JetBrains products. Currently, RustRover is in [EAP](https://www.jetbrains.com/resources/eap/), making it unstable from a registry key point of view. 

You can't currently use the `registryKeysForJetBrainsIDE` for RustRover!